addons/l10n_ar/models/account_move.py

```diff
@@ -223,7 +223,7 @@ def _get_formatted_sequence(self, number=0):
     def _get_starting_sequence(self):
         """ If use documents then will create a new starting sequence using the document type code prefix and the
         journal document number with a 8 padding number """
-        if self.journal_id.l10n_latam_use_documents and self.env.company.country_id.code == "AR":
+        if self.journal_id.l10n_latam_use_documents and self.company_id.account_fiscal_country_id.code == "AR":
```

Comment:
use company_id.country_code maybe

Reply:
this is both in V14 and V15 = country_code

Reply:
@zaoral @jjscarafia re-target this to V14 instead of V15.
Use the following in V14
`self.company_id.country_id.code == "AR"`